chatterbox/src/Main/Header/index.css

```diff
@@ -0,0 +1,36 @@
+.MuiToolbar-root-39, .MuiToolbar-gutters-40 {
+
+	height: 103px;
+	width:  1440px; 
+    background-color: #01B9BD;
+}
+ 
+div.MuiToolbar-root-40.MuiToolbar-gutters-41 {
+    background-color: #01B9BD;
+
+} 
+button.MuiButtonBase-root-56.MuiButton-root-41.MuiButton-raised-48 {
```

Comment:
why do you have 48 and what is root 41? 
bad naming

Reply:
Is this a Git Submodule component?

Reply:
Shouldn't it be somethng that is install with npm?

Reply:
@jaderabbit no I figured it out it's just an issue with the project structure they ran the `create-react-app chatterbox` in the root folder of the app folder instead of the root folder of their projects folder